registry.json

```diff
@@ -371,6 +373,7 @@
   "jsnlog": "npm:jsnlog",
   "jsol": "github:daepark/JSOL",
   "jsplumb": "github:sporritt/jsPlumb",
+  "jspm-loader-css":"npm:jspm-loader-css",
```

Comment:
I'm not sure this is still supported? Is it still working in jspm 0.16? I'm not sure it does in jspm 0.17?